src/test/unit/views/summary-of-publications.test.ts

```diff
@@ -67,4 +68,28 @@ describe('Summary of publications page', () => {
             expect(body[4].innerHTML).contains('Sorry, no lists found for this court', 'Non SJP is not displayed');
         });
     });
+
+    describe('Non SJP Summary of Pubs without locationId query parameter', () => {
```

Comment:
I'm thinking we should create a separate error view test for this, as although using the summary of pubs to trigger it, it's specifically the error view

Reply:
I have moved these 2 tests out to error.test.ts. I am still using summary of publications page on the error test as this error page hasn't got its owned url.